packages/mui-base/src/useButton/useButton.ts

```diff
@@ -235,5 +235,6 @@ export default function useButton(parameters: UseButtonParameters): UseButtonRet
     setFocusVisible,
     disabled,
     active,
+    ref: handleRef,
```

Comment:
I realized that it was impossible to use two hooks (`useListItem` and `useButton`) on the same element, as the refs returned by the hooks' `getRootProps` would overwrite one another. Having a ref returned separately allows us to combine them.